Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "artichoke"
-version = "0.1.0-pre.0"
+version = "0.2.0-pre.0"
```

Comment:
thanks for bumping the crate version, but I think we bumped the wrong one 🤣 

Can you back out this change and bump the version in `mezzaluna-feature-loader` to 0.2.0?

https://github.com/artichoke/artichoke/blob/7924a8eb304deb8d82c8c434590e7120943d97cf/mezzaluna-feature-loader/Cargo.toml#L3